package-lock.json

```diff
@@ -1182,192 +1161,26 @@
-          "resolved": "https://registry.npmjs.org/google-auth-library/-/google-auth-library-6.0.3.tgz",
-          "integrity": "sha512-2Np6ojPmaJGXHSMsBhtTQEKfSMdLc8hefoihv7N2cwEr8E5bq39fhoat6TcXHwa0XoGO5Guh9sp3nxHFPmibMw==",
-          "requires": {
-            "arrify": "^2.0.0",
-            "base64-js": "^1.3.0",
-            "ecdsa-sig-formatter": "^1.0.11",
-            "fast-text-encoding": "^1.0.0",
-            "gaxios": "^3.0.0",
-            "gcp-metadata": "^4.1.0",
-            "gtoken": "^5.0.0",
-            "jws": "^4.0.0",
-            "lru-cache": "^5.0.0"
-          }
-        },
-        "google-p12-pem": {
-          "version": "3.0.1",
-          "resolved": "https://registry.npmjs.org/google-p12-pem/-/google-p12-pem-3.0.1.tgz",
-          "integrity": "sha512-VlQgtozgNVVVcYTXS36eQz4PXPt9gIPqLOhHN0QiV6W6h4qSCNVKPtKC5INtJsaHHF2r7+nOIa26MJeJMTaZEQ==",
-          "requires": {
-            "node-forge": "^0.9.0"
-          }
-        },
-        "gtoken": {
-          "version": "5.0.1",
-          "resolved": "https://registry.npmjs.org/gtoken/-/gtoken-5.0.1.tgz",
-          "integrity": "sha512-33w4FNDkUcyIOq/TqyC+drnKdI4PdXmWp9lZzssyEQKuvu9ZFN3KttaSnDKo52U3E51oujVGop93mKxmqO8HHg==",
-          "requires": {
-            "gaxios": "^3.0.0",
-            "google-p12-pem": "^3.0.0",
-            "jws": "^4.0.0",
-            "mime": "^2.2.0"
-          }
-        },
-        "mime": {
-          "version": "2.4.6",
-          "resolved": "https://registry.npmjs.org/mime/-/mime-2.4.6.tgz",
-          "integrity": "sha512-RZKhC3EmpBchfTGBVb8fb+RL2cWyw/32lshnsETttkBAyAUXSGHxbEJWWRXc751DrIxG1q04b8QwMbAwkRPpUA=="
-        },
-        "p-limit": {
-          "version": "3.0.1",
-          "resolved": "https://registry.npmjs.org/p-limit/-/p-limit-3.0.1.tgz",
-          "integrity": "sha512-mw/p92EyOzl2MhauKodw54Rx5ZK4624rNfgNaBguFZkHzyUG9WsDzFF5/yQVEJinbJDdP4jEfMN+uBquiGnaLg==",
-          "requires": {
-            "p-try": "^2.0.0"
-          }
-        },
-        "readable-stream": {
-          "version": "3.6.0",
-          "resolved": "https://registry.npmjs.org/readable-stream/-/readable-stream-3.6.0.tgz",
-          "integrity": "sha512-BViHy7LKeTz4oNnkcLJ+lVSL6vpiFeX6/d3oSH8zCW7UxP2onchk+vTGB143xuFjHS3deTgkKoXXymXqymiIdA==",
-          "requires": {
-            "inherits": "^2.0.3",
-            "string_decoder": "^1.1.1",
-            "util-deprecate": "^1.0.1"
-          }
-        },
-        "teeny-request": {
-          "version": "7.0.0",
-          "resolved": "https://registry.npmjs.org/teeny-request/-/teeny-request-7.0.0.tgz",
-          "integrity": "sha512-kWD3sdGmIix6w7c8ZdVKxWq+3YwVPGWz+Mq0wRZXayEKY/YHb63b8uphfBzcFDmyq8frD9+UTc3wLyOhltRbtg==",
-          "requires": {
-            "http-proxy-agent": "^4.0.0",
-            "https-proxy-agent": "^5.0.0",
-            "node-fetch": "^2.2.0",
-            "stream-events": "^1.0.5",
-            "uuid": "^8.0.0"
+        "aws-sdk": {
```

Comment:
`aws-sdk` is installed twice now.

Reply:
I moved `aws-sdk` to a dev dependency as well, and made it match the version in object-persistor.

It's listed once as a non-dev-dependency in object-persistor, and once as a dev-dependency. The integritiy hashes are the same, so I think this ok?

Reply:
They seem to be different tho, possibly using different nested dependencies:

<details>

``` 
dev-environment$ make filestore/{git_checkout,git_pull,install} BRANCH=spd-remove-gcs-module
cd filestore && git checkout spd-remove-gcs-module
Switched to branch 'spd-remove-gcs-module'
Your branch is up to date with 'origin/spd-remove-gcs-module'.
cd filestore && git pull
Already up to date.
bin/run -T --no-deps filestore npm install -q --no-save
npm WARN @typescript-eslint/parser@1.13.0 requires a peer of eslint@^5.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN filestore-sharelatex@0.1.4 No license field.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 28 packages from 16 contributors, removed 16 packages, updated 6 packages and audited 822 packages in 7.273s

30 packages are looking for funding
  run `npm fund` for details

found 25 low severity vulnerabilities
  run `npm audit fix` to fix them, or `npm audit` for details
touch filestore/node_modules/.guard

dev-environment$ find filestore/node_modules/ -name aws-sdk | xargs du -h -d 0
53M     filestore/node_modules/aws-sdk
52M     filestore/node_modules/@overleaf/object-persistor/node_modules/aws-sdk
```

^ 53MB on top-level and 52MB nested in object-persistor
</details>

Reply:
(I would have guessed that the top-level dependency could share more dependencies with the rest, but it's the object-persistor one with fewer nested dependencies :shrug:)

Reply:
```
-> find filestore/node_modules/ -name aws-sdk | xargs diff -qr
Only in filestore/node_modules/aws-sdk: node_modules
Files filestore/node_modules/@overleaf/object-persistor/node_modules/aws-sdk/package.json and filestore/node_modules/aws-sdk/package.json differ
```

```
-> diff filestore/node_modules/@overleaf/object-persistor/node_modules/aws-sdk/package.json filestore/node_modules/aws-sdk/package.json
2c2
<   "_from": "aws-sdk@https://registry.npmjs.org/aws-sdk/-/aws-sdk-2.718.0.tgz",
---
>   "_from": "aws-sdk",
6c6
<   "_location": "/@overleaf/object-persistor/aws-sdk",
---
>   "_location": "/aws-sdk",
9,10c9,11
<     "type": "remote",
<     "raw": "aws-sdk@https://registry.npmjs.org/aws-sdk/-/aws-sdk-2.718.0.tgz",
---
>     "type": "tag",
>     "registry": true,
>     "raw": "aws-sdk",
13,15c14,16
<     "rawSpec": "https://registry.npmjs.org/aws-sdk/-/aws-sdk-2.718.0.tgz",
<     "saveSpec": "https://registry.npmjs.org/aws-sdk/-/aws-sdk-2.718.0.tgz",
<     "fetchSpec": "https://registry.npmjs.org/aws-sdk/-/aws-sdk-2.718.0.tgz"
---
>     "rawSpec": "",
>     "saveSpec": null,
>     "fetchSpec": "latest"
18c19,20
<     "/@overleaf/object-persistor"
---
>     "#DEV:/",
>     "#USER"
22,23c24,25
<   "_spec": "aws-sdk@https://registry.npmjs.org/aws-sdk/-/aws-sdk-2.718.0.tgz",
<   "_where": "/app",
---
>   "_spec": "aws-sdk",
>   "_where": "/home/simon/overleaf/sharelatex-dev-environment/filestore",
```
I am reluctant to go down the rabbit hole to figure out why this is. I think it's probably fine.